homeassistant/components/homekit/type_thermostats.py

```diff
@@ -125,6 +126,7 @@
     CURRENT_HVAC_COOL: HC_HEAT_COOL_COOL,
     CURRENT_HVAC_DRY: HC_HEAT_COOL_COOL,
     CURRENT_HVAC_FAN: HC_HEAT_COOL_COOL,
+    CURRENT_HVAC_AUTO: HC_HEAT_COOL_AUTO,
```

Comment:
This is already supported by HomeKit: https://developer.apple.com/documentation/homekit/hmcharacteristicvalueheatingcooling

Reply:
Please open another PR to make changes to homekit.  We don't want to combine changes with another integration in the same PR.

Also we need a test for all homekit changes as we have 100% coverage on all HomeKit accessory code